{"name": "test_login_error[item0]", "status": "passed", "parameters": [{"name": "item", "value": "{'username': '888888', 'pwd': '655444', 'code': '123456', 'excepted': '抱歉，账号或密码错误！'}"}], "start": 1558343730961, "stop": 1558343732547, "uuid": "c8da7273-839b-4cf4-937d-71aa4cfb5bde", "historyId": "0d871edcd89274fbac7338acf71ded99", "fullName": "TestCase.test_0_login.TestLogin#test_login_error[item0]", "labels": [{"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_0_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "bj_jd-PC"}, {"name": "thread", "value": "6240-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_0_login"}]}